# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/iproute2/hotfix-basename.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- iproute2-6.9.0/rdma/rdma.c.vanilla	2024-07-25 13:03:58.494781449 +0200
+++ iproute2-6.9.0/rdma/rdma.c	2024-07-25 13:04:21.529781171 +0200
@@ -4,6 +4,7 @@
  * Authors:     Leon Romanovsky <leonro@mellanox.com>
  */
 
+#include <libgen.h>
 #include "rdma.h"
 #include "version.h"
 #include "color.h"
